README.md

```diff
@@ -27,7 +27,6 @@ This is the legacy rich text editor for the Wagtail CMS. Based on [Hallo.js](htt
 
 ## Installing the Hallo Editor
 
-- Important: requires jQuery and jQueryUI, which are not included and may not always be included with Wagtail.
```

Comment:
I have removed this because I think it’s confusing – at the moment there’s nothing to do, and when Wagtail does remove those dependencies, it’ll be a maintainer task to add them here or do something else.